submissions/pseudoscience/pseudoscience_non-fiction.md

```diff
@@ -0,0 +1,107 @@
+## A scientific examination of the Factorio world
+
+Suspension of disbelief is a well-known concept, however that does not stop me from ignoring it completely. Some days ago I saw a [Reddit post](https://www.reddit.com/r/factorio/comments/jaz6yl/calculating_the_density_of_nauvis/) about calculating the density of Nauvis, and that gave me an idea of making a just-for-fun compilation of all the ridiculous science that goes on in the Factorio universe.
```

Comment:
> this has been a gigiantic nerd circle j*** from the beginning
I feel like this intro needs to more embody this message you wrote in discord earlier. We are hear to dive into an unnecesary amount of nerdy detail about the weird physics of the game we love and the intro should represent that better.